contrib/syslog-debun

```diff
@@ -152,12 +177,17 @@ done
 [ -n "$timeout" ] && [ -z "$debug_mode" ] && debun_usage 2 "Timeout without debug mode or packet capture"
 [ -n "$privacy_mode" ] && [ "x$debug_params" = "x$default_debug_params" ] && debug_params="$default_ldebug_params"
 #if [ -n "$pcap_params" ]; then
-#	hash tcpdump || debun_usage 2 "Packet capture requested, but no tcpdump in your PATH"
+#	hash tcpdump 2>/dev/null || debun_usage 2 "Packet capture requested, but no tcpdump in your PATH"
 #fi
 syslogbin=${binprefix}/sbin/syslog-ng
-syslogrealbin=${binprefix}/libexec/syslog-ng
```

Comment:
Are you sure, it's a good idea to remove that?
My memories are foggy about that part, but I have some foggy memories, that maybe on solaris versions, the syslog binary was wrapped by a simple shell script in sbin, and the actual binary was inside libexec, so ldd should test that.